platforms/corda-ent/charts/float/files/firewall.conf

```diff
@@ -0,0 +1,37 @@
+firewallMode: FloatOuter
+
+# Uses /opt/corda/workspace as base directory
+networkParametersPath: "network-parameters"
+
+inboundConfig: {
+  listeningAddress: "{{ .Values.dmz.external}}:{{ .Values.node.p2pPort}}"
```

Comment:
spaces needed "  }}"

Reply:
Fixed in [356b94a](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/893/commits/356b94a6b9bda311911f1dc595f0acb84792baa8)